packages/components/src/components/codeChunk/codeChunk.tsx

```diff
@@ -27,6 +27,7 @@ export class CodeChunkComponent implements CodeComponent<CodeChunk> {
   private static readonly slots = {
     text: 'text',
     outputs: 'outputs',
+    caption: 'caption',
```

Comment:
Just to note that at least `label` and `errors` will be required here to eventually https://help.stenci.la/references/schema/CodeChunk. You may want to add `errors` later, in a separate PR, but `label` is almost always associated with a `caption` so if seems like it would be best to add it as part of this PR.